### ISSUE TITLE
Travis verification no longer adds state to the pull request and sometimes fails on "apt-get -qq update"

### ISSUE THREAD

**Issue by gurevichmark** [xcat2/xcat-core#6092]
Two Travis issues:

1. Back in June 2018, Travis build verification used to append this information to each pull request:

![image](https://user-images.githubusercontent.com/16106630/54311404-e168a800-45aa-11e9-9ae4-a2d11961b8eb.png)

That seems to be no longer working.
Looking at the Travis log, I see these errors, which I think prevent Travis from adding status to the pull request:

```
[send_back_comment] method = POST to https://api.github.com/repos/xcat2/xcat-core/issues/6074/comments
Use of uninitialized value $ENV{"xcatbotuser"} in concatenation (.) or string at ./travis.pl line 263.
Use of uninitialized value in concatenation (.) or string at ./travis.pl line 263.
```

2. I have noticed several failures of `apt-get -qq update` command during the "install_xcat" phase of Travis run. Need to investigate the cause:
![image](https://user-images.githubusercontent.com/16106630/54312759-162a2e80-45ae-11e9-8412-b896e1777f60.png)

**Comment by robin2008** [comment]
@hu-weihua

**Comment by gurevichmark** [comment]
@hu-weihua I would like to work on this issue.
Can you help me understand where and how `$ENV{"xcatbotuser"}` and `$ENV{"xcatbotpw"}` get set ?

**Comment by hu-weihua** [comment]
@gurevichmark, this is a long story. 

The original work process for sending back CI test result to git web page was set user name and password in CI configuration page. But it resulted in a security problem which logged in https://github.ibm.com/xcat2/jenkins/issues/171.

The root cause is written in  https://github.ibm.com/xcat2/jenkins/issues/176.

We still have an opening task https://github.com/xcat2/xcat2-task-management/issues/280 to resolve this problem. But I have not gotten time for this. If you have time to investigate how to make it work. I appreciate very much.

**Comment by gurevichmark** [comment]
@hu-weihua Thank you for the explanation. Maybe it is too much work to fix this.

What if instead of writing the result on the travis run into the PR, we just display the success or failure message into the travis output log. This way if verification fails, the travis run will be marked with red `x` in the PR and each owner will need to click on the `Details` button to display the log and see the reason for the failure.

**Comment by hu-weihua** [comment]
@gurevichmark, I have merged PR #6213, thanks for all your help.
